CHANGELOG.md

```diff
@@ -14,3 +14,15 @@
 
 ## [0.0.0 - 0.0.1]
 - Reference implementation for the PureSentinel
+
+## [0.0.1 - 0.0.2] Implement the three sentinels
```

Comment:
can change `0.0.2` to `0.1.0` since this lib is currently at `0.1.0`. Can we keep version order descending too please